criu/apparmor.c

```diff
@@ -367,11 +472,13 @@ int collect_aa_namespace(char *profile)
 		pos++;						\
 	}
 
-static int write_aa_policy(AaNamespace *ns, char *path, int offset, char *rewrite)
+static int write_aa_policy(AaNamespace *ns, char *path, int offset, char *rewrite, bool suspend)
 {
 	int i, my_offset, ret;
 	char *rewrite_pos = rewrite, namespace[PATH_MAX];
 
+	BUG_ON(rewrite && suspend);
```

Comment:
@0x7f454c46 opened this ticket some time ago: https://github.com/checkpoint-restore/criu/issues/767

Reply:
Ok, removing the BUG_ON.